app/models/user.rb

```diff
@@ -6,6 +6,10 @@ class User < ApplicationRecord
   validates :name, presence: true, length: { maximum: 50 }
   validates :posts_counter, numericality: { greater_than_or_equal_to: 0, only_integer: true }
 
+  # add devise
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job working on this milestone. For your app to ask confirmation of email, `:confirmable` has to be enabled in the `user` model.
```
	devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
```

Reply:
@t-yanick 
i did add it to the user using the migrate

![email to user](https://user-images.githubusercontent.com/110314309/233550931-c6f48e84-262f-4bf8-9402-918840bf84c6.JPG)